packages/core/src/contents/utils.js

```diff
@@ -98,6 +98,15 @@ export const getPageRanking = metadata => {
   return ranking;
 };
 
+const getCommercePagesRanked = result => {
```

Comment:
Add here some JsDoc documentation with the purpose of this method.

Reply:
done